.banner.txt

```diff
@@ -1,6 +1,7 @@
 /*!
  * <%= pkg.name %> library v<%= pkg.version %>
  * <%= pkg.homepage %>
+ * @preserve
```

Comment:
didn't know that one, makes sense!

PS isn't it 3:53 your time, what the hell are you doing on github at that time?! :)